gen/internals.py

```diff
@@ -284,6 +309,13 @@ class SkipError(Exception):
     pass
 
 
+# has a deterministic chunk before and after the variable name so we can deterministically get out
+# the name
+LATE_BIND_PLACEHOLDER_START = 'LATE_BIND_PLACEHOLDER_START_'
```

Comment:
`PLACEHOLDER` usually means they are substituted /replaced. I guess, it is technically right. It is more than just a placeholder. It's a `MARKER` or `DELIMITER`.  Would those terms be more preferable instead of placeholder?